apps/versions/tests.py

```diff
@@ -94,6 +94,21 @@ def test_dict_from_int():
     eq_(d['pre_ver'], 2)
 
 
+@pytest.mark.parametrize("addon_type", amo.GROUP_TYPE_ADDON)
```

Comment:
I would never even have considered spelling `parameter` as `parametre` which makes `parametrize` look really weird. Shouldn't it use an `s` if it is using the `re` version also? What a strange language.